addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -1,130 +1,35 @@
-                        <t t-set="seats_max" t-value="min(seats_max_ticket, seats_max_event)"/>
-                        <t t-foreach="range(0, seats_max)" t-as="nb">
-                            <option t-esc="nb" t-att-selected="len(tickets) == 1 and nb == 1 and 'selected'"/>
-                        </t>
-                    </select>
-                </t>
+    <!-- Add price information on tickets (multi tickets, aka in collapse) -->
+    <xpath expr="//div[hasclass('o_wevent_registration_multi_select')]" position="before">
+<!--         <t t-if="ticket.price or editable">
+            <t t-if="(ticket.price-website.get_current_pricelist().currency_id._convert(ticket.price_reduce, event.company_id.sudo().currency_id, event.company_id, datetime.date.today())) &gt; 1 and website.get_current_pricelist().discount_policy == 'without_discount'">
+                <del class="text-danger mr-1" t-field="ticket.price" t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.get_current_pricelist().currency_id}"/>
             </t>
+            <span t-field="ticket.price_reduce" t-options="{'widget': 'monetary', 'display_currency': website.pricelist_id.currency_id}" groups="account.group_show_line_subtotals_tax_excluded"/>
+            <span t-field="ticket.price_reduce_taxinc" t-options="{'widget': 'monetary', 'display_currency': website.pricelist_id.currency_id}" groups="account.group_show_line_subtotals_tax_included"/>
+            <span itemprop="price" class="d-none" t-esc="ticket.price"/>
+            <span itemprop="priceCurrency" class="d-none" t-esc="website.pricelist_id.currency_id.name"/>
         </t>
+        <span t-if="not ticket.price and not editable" class="font-weight-bold text-uppercase">Free</span> -->
     </xpath>
-    <xpath expr="//form[@id='registration_form']" position="attributes">
-        <attribute name="t-if">event.event_registrations_open and not all([ticket.is_expired for ticket in event.event_ticket_ids])</attribute>
-    </xpath>
-    <xpath expr="//form[@id='registration_form']" position="before">
-        <div t-if="event.start_sale_date and event.start_sale_date &gt; datetime.date.today() and not event.event_registrations_open" class="alert alert-info mb-5" role="status">
-            <em >Ticket  Sales starting on <t t-esc="event.start_sale_date"/></em>
+    <!-- Add price information on tickets (mono ticket, aka not in collapse) -->
+    <xpath expr="//div[hasclass('o_wevent_registration_single')]//h6" position="after">
+        <div class="border-right px-3">
+            <t t-if="tickets.price or editable">
+                <t t-if="(tickets.price-website.get_current_pricelist().currency_id._convert(tickets.price_reduce, event.company_id.sudo().currency_id, event.company_id, datetime.date.today())) &gt; 1 and website.get_current_pricelist().discount_policy == 'without_discount'">
```

Comment:
`ticket.price-website..get_current_pricelist()`

Didn't even know you could have "-" in variables...
(I know you just moved the code ^^)